### PR TITLE
Add the new branch 1.6 in containerd to pull-containerd-node-e2e

### PR DESCRIPTION
**Pull request by dims**
Signed-off-by: Davanum Srinivas <davanum@gmail.com>

### CODE REVIEW

**Review by endocrimes — APPROVED** [review]
/lgtm
/hold
feel free to unhold when ready to merge, giving a chance for containerd folks to take a look

File: config/jobs/containerd/containerd/containerd-presubmit-jobs.yaml [hunk 1]
```diff
@@ -31,6 +31,7 @@ presubmits:
     branches:
     - main
     - release/1.5
+    - release/1.6
```
Comment (mikebrow) [inline config/jobs/containerd/containerd/containerd-presubmit-jobs.yaml:34]: also up on line 7
`- release/1.6`

Comment (dims) [reply config/jobs/containerd/containerd/containerd-presubmit-jobs.yaml:35]: Done!

**Review by mikebrow — APPROVED** [review]
LGTM
Thanks!

**Pull request merged**